.github/README.md

```diff
@@ -0,0 +1,31 @@
+<!-- TITLE -->
```

Comment:
Is the README not generally placed in the root, so it can be easy to locate, do we have a reason to deviate from the convention?

Reply:
Because this repo is on github, it's a make a cleaner first look on the page. The .github directory is also used if we make issue templates. But since we're not using those yet, we can move the README back into the root.